eventhub/endpoint.bal

```diff
@@ -18,18 +18,28 @@ import ballerina/http;
 import ballerina/lang.'xml as xmllib;
 import ballerina/regex;
 
-# Client for Azure Event Hub connector.
+# Ballerina Azure Event Hub connector provides the capability to access Azure Event Hubs REST API.
+# Azure Event Hubs is a highly scalable data ingress service that ingests millions of events per second 
```

Comment:
so that you can-> allowing you to

Reply:
massive -> large amount

Reply:
"Azure Event Hubs is a highly scalable data ingress service that ingests millions of events per second, allowing you to process and analyze the large amounts of data produced by your connected devices and applications. 	 
"